pkg/apis/scaler/v1alpha1/crd_demand.go

```diff
@@ -116,9 +120,10 @@ var (
 											Type:     "object",
 											Required: []string{"count", "cpu", "memory"},
 											Properties: map[string]v1beta1.JSONSchemaProps{
-												"count":  {Type: "integer", Minimum: &oneFloat},
-												"cpu":    {Type: "string", MinLength: &oneInt},
-												"memory": {Type: "string", MinLength: &oneInt},
+												"count":         {Type: "integer", Minimum: &oneFloat},
+												"cpu":           {Type: "string", MinLength: &oneInt},
+												"memory":        {Type: "string", MinLength: &oneInt},
+												"is-long-lived": {Type: "boolean"},
```

Comment:
this is incorrect unfortunately.
the new field is at the same level as `"units", "instance-group"`